src/components/templates/ConversationSpace/constants.ts

```diff
@@ -1,71 +1,73 @@
+import { DEFAULT_TABLE_CAPACITY } from "settings";
+
 import { Table } from "types/Table";
 
 export const TABLES: Table[] = [
```

Comment:
This should be refactored to use `generateTables` in a similar way to how it's done in `src/components/templates/Jazzbar/JazzTab/constants.ts`

Reply:
fixed in 628728c978f8946518261612288a3324163ca176